spec/lib/seed/facility_seeder_spec.rb

```diff
@@ -1,6 +1,12 @@
 require "rails_helper"
 
 RSpec.describe Seed::FacilitySeeder do
+  it "creates a protocol" do
```

Comment:
Could also be worth confirming that a Protocol has been created in the db, similar to the facilities/groups test immediately following this.

Reply:
The protocol creation in the DB is tested in the `protocol_seeder_spec`, so this spec just tests the right collaboration with that class. I usually take this approach to avoid double-testing the same implementations over and over. What do you think?

Reply:
Sounds good to me. I'm reviewing this with little context, and I generally will agree with you about testing boundaries.

Reply:
No worries at all, keep em coming 👌